pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/Policies.java

```diff
@@ -107,7 +107,7 @@
     public SchemaCompatibilityStrategy schema_compatibility_strategy = SchemaCompatibilityStrategy.UNDEFINED;
 
     @SuppressWarnings("checkstyle:MemberName")
-    public Boolean is_allow_auto_update_schema = null;
+    public Boolean is_allow_auto_update_schema = true;
```

Comment:
We can't set it to true because the broker level config might be false

Reply:
I will revert this